app/templates/components/list-filter.hbs

```diff
@@ -1,2 +1,4 @@
-City: {{input value=value key-up=(action 'handleFilterEntry')}}
+<label for="filterByCity" class="screen-reader">Filter By City</label>
+  {{input id="filterByCity" value=value key-up=(action 'handleFilterEntry') class="light" placeholder="Filter By City" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="rental-list"}}
```

Comment:
I thought that too and tried with initial lack of success.  will take another look. thx